doc/classes/Node2D.xml

```diff
@@ -1,10 +1,11 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="Node2D" inherits="CanvasItem" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		A 2D game object, inherited by all 2D-related nodes. Has a position, rotation, scale, and skew.
+		Base object in 2D space, inherited by all 2D nodes.
 	</brief_description>
 	<description>
-		A 2D game object, with a transform (position, rotation, and scale). All 2D nodes, including physics objects and sprites, inherit from Node2D. Use Node2D as a parent node to move, scale and rotate children in a 2D project. Also gives control of the node's render order.
+		The [Node2D] node is the base representation of a node in 2D space. All other 2D nodes inherit from this class.
```

Comment:
This is for consistency with https://github.com/godotengine/godot/pull/87440 . 
However, in hindsight, I can't help but notice loss of information. I personally think it's fine, given the very generic and oddly written nature of them (_"also... gives... control of..."_).

Reply:
Yeah, imo the previous version is better here, this change is in danger of removing all the "documentation" from the documentation...

Reply:
I elaborate more on types that inherit Node2D. The rest is... true, but it actually applies to **CanvasItem** as a whole, and it feels wrong to mention it here, especially because of the `[b]Note:[/b] Since both [Node2D] and [Control] inherit from [CanvasItem],` below.